README.md

```diff
@@ -84,13 +78,12 @@ juju deploy ./bundle.yaml
 proxy configuration options as described in the Proxy configuration section
 above.
 
-This bundle exposes the kubeapi-load-balancer and kibana charms by default
-This means those charms are accessible through their public addresses.
+This bundle exposes the kubeapi-load-balancer charm by default, so it is
```

Comment:
I think the kubernetes-worker charm is also exposed by default for ingress stuff. Can you check this and add worker if necessary?